osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -31,12 +31,14 @@ protected override void LoadComplete()
             parentInputManager = GetContainingInputManager();
         }
 
+        // IMPORTANT: Do not change the order of key bindings in this list.
+        // It is used to decide the int values when storing settings in DatabasedKeyBindingContainer.
```

Comment:
Wait so they're not based on the int value of the settings themselves? Doesn't this mess up all these bindings then, because the members inside `OverlayKeyBindings` were previously at the very start of the enumerable.

Reply:
You have a point. Let me check on that, there must be something else going on here.

Reply:
It turns out ordering *is* important, but for a different reason. I'll revert this change and update the comment. The actual failure is elsewhere, as discussed on discord.